ui/cap-react/src/antd/admin/formComponents/widgets/index.js

```diff
@@ -1,7 +1,7 @@
-import Select from "./SelectWidget";
+import SelectColumnsWidget from "./SelectColumnsWidget";
 
 const widgets = {
-  selectColumns: Select,
+  selectColumns: SelectColumnsWidget,
```

Comment:
why do we need this actually? cant we do it with a simple select (or other widget), we can remove the `SelectColumnsWidget`, no?

Reply:
e.g. similar result:
```
            span: {
              type: "string",
              title: "Field Width",
              enum: [
                6,
                8,
                12,
                16,
                18,
                24,
              ],
              enumNames: [
                "25%",
                "33%",
                "50%",
                "66%",
                "75%",
                "100%",
              ]
            },
```

Reply:
It can be done directly from the schema without a custom field, but the problem with that is that we would have to copy-paste that chunk in basically all the fields in `fieldTypes.js`. Now that we talk about this, it would actually be a good idea to have a task to externalize some of the JSON chunks that repeat for every field and do some cleanup of that file. Something like:

```js
const baseUiOptions: { /* the common stuff such as the span settings */ }

checkbox: {
   optionsUiSchemaUiSchema: {
        "ui:options": { 
            ...baseUiOptions,
            //whetever else
        }
    }
}
```

Reply:
Either we can do it in this PR, or as next task. 
We can either:
- just inject/merge the default chunk in every PropertyKeyEditor components, when we get the schema/uiSchema from `fieldTypes.js`
- use definitions from JSONschemas and then reference to them 
```
{
  "definitions": {
    "address": {
      "type": "object",
      "properties": {
        "street_address": {
          "type": "string"
        },
        "city": {
          "type": "string"
        }
      },
      "required": [
        "street_address",
        "city"
      ]
    }
  },
  "type": "object",
  "properties": {
    "billing_address": {
      "title": "Billing address",
      "$ref": "#/definitions/address"
    },
}
```
- as variable (maybe better the `definition`)
- something else?

Lets discuss